src/BenchmarksClient/Startup.cs

```diff
@@ -151,16 +151,11 @@ private static Process StartProcess(ClientJob job)
                 }
             }
 
-            if (job.PipelineDepth > 0)
-            {
-                command += $" -s scripts/pipeline.lua";
-            }
-
-            command += $" {job.ServerBenchmarkUri}";
+            command += $" -s scripts/pipeline.lua {job.ServerBenchmarkUri} -- {job.Method}";
```

Comment:
Right now we don't have any scenarios that need a different method and no pipelining so I'm not going to create another lua script for it and rely on the modifications to the pipeline.lua script. If we eventually do have a scenario like this, we'll need to create a different script and tweak this logic.